lib/json_api_controller/legacy_policy.rb

```diff
@@ -0,0 +1,17 @@
+module JsonApiController
+  module LegacyPolicy
```

Comment:
Not a fan of the `LegacyPolicy` name but i'm having a hard time coming up with another one, perhaps `RoledPolicy` to link the `RoledControllerPolicy` objects as well via name.